charts/charts/mysql/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
Could We use the existing mysql charts，example https://artifacthub.io/packages/helm/bitnami/mysql